go.mod

```diff
@@ -15,12 +15,11 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.6.1
 	github.com/spf13/pflag v1.0.5
-	github.com/submariner-io/admiral v0.14.3
-	github.com/submariner-io/cloud-prepare v0.14.3
-	github.com/submariner-io/lighthouse v0.14.3
-	github.com/submariner-io/shipyard v0.14.3
-	github.com/submariner-io/submariner v0.14.3
-	github.com/submariner-io/submariner-operator v0.14.4-0.20230317175327-1161aa7281d7
+	github.com/submariner-io/admiral v0.13.3
```

Comment:
Can you please check why admiral and some of the other components are pointing to 0.13.3?

Reply:
Ack, thanks.
Created https://github.com/submariner-io/subctl/pull/718 manually